spec/spec.opts

```diff
@@ -1,6 +1,5 @@
 --format
-s
+documentation
 --colour
---loadby
```

Comment:
I added the parallel_spec thing to parallelise the tests. It doesn't work with `loadby` though.